src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -19,6 +19,9 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_full_solution_analysis}" />
                     <CheckBox x:Name="Enable_navigation_to_decompiled_sources"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_navigation_to_decompiled_sources}" />
+                    <CheckBox x:Name="Enable_use_nullable_reference_types"
+                              IsThreeState="True"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_navigation_to_decompiled_sources}" />
```

Comment:
Content is the same as the previous line?

Reply:
Ugh, copy paste fail on my laptop mid meeting :)